doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the particle inserted within the previous step. We will discuss the different parameter file sections. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. Most subsections are explained in detail in other CFD-DEM examples such as:  `Gas-solid spouted bed <../../../examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for :math:`2` s with a time step of :math:`0.005` s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
+      set output frequency     = 20
+      set time end             = 2
+      set time step            = 0.005
+      set output path          = ./output/
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. We choose a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water.
+
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.0000008379
+        set density             = 996.8
+      end
+    end
+
+Initial Conditions
+~~~~~~~~~~~~~~~~~~
+
+We choose zero initial conditions for the velocity.
+
+.. code-block:: text
+
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+        set Function expression = 0; 0; 0; 0
+      end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on all the walls of the channel and the channel except the bottom and the top of the channel where a no-slip boundary condition is imposed. For more information about the boundary conditions, please refer to the `Boundary Conditions Section <../../../parameters/cfd/boundary_conditions_cfd.html>`_
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 6
+      subsection bc 0
+        set id   = 0
+        set type = slip
+      end
+      subsection bc 1
+        set id   = 1
+        set type = slip
+      end
+      subsection bc 2
+        set id   = 2
+        set type = noslip
+      end
+      subsection bc 3
+        set id   = 3
+        set type = noslip
+      end
+      subsection bc 4
+        set id   = 4
+        set type = slip
+      end
+      subsection bc 5
+        set id   = 5
+        set type = slip
+      end
+    end
```

Comment:
Why?
Just put slip everywhere, it seems weird to have no slip at the top and the bottom. In the end you want to compare falling in a forever container kinda. The wall should have no effect on your simulation.